src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Helpers/Parser/BinderInvocation.cs

```diff
@@ -27,5 +28,72 @@ internal sealed record BinderInvocation
                 Location = invocationSyntax.GetLocation()
             };
         }
+
+        private static bool IsCandidateInvocationExpressionSyntax(SyntaxNode node, out InvocationExpressionSyntax? invocationSyntax)
```

Comment:
Perhaps indicate with a comment that this is a pre-filtering stage intended to avoid looking up as many invocation symbols as possible before the final filtering logic is done?

Reply:
Addressed your feedback in https://github.com/dotnet/runtime/pull/90619.